packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -0,0 +1,231 @@
+  const [messages, setMessages] = useState<IMessage[]>([]);
+  const content = useRef<HTMLDivElement>(null);
+  const input = useRef<HTMLTextAreaElement>(null);
+  const footer = useRef<HTMLDivElement>(null);
+  const classes = chatWidgetStyles();
+
+  useEffect(() => {
+    content.current?.lastElementChild?.scrollIntoView();
+  }, [messages]);
+
+  useEffect(() => {
+    async function fetchMessages() {
+      const session = authHook.session();
+      if (session !== null) {
+        console.log(patientId);
+        console.log(teamId);
+        console.log(userId);
+        const messages = await getChatMessages(session, teamId, patientId);
+        setMessages(messages);
+      }
+    }
+    fetchMessages();
+  }, [authHook, userId, patientId, teamId]);
+
+  const onEmojiClick = (_event: React.MouseEvent, emojiObject: IEmojiData) => {
+    setShowPicker(false);
+    setInputText(inputText + emojiObject.emoji);
+  };
+
+  const resetInputSize = () => {
+    if (!content.current || !footer.current || !input.current) {
+      throw new Error("Cannot find elements for resize");
+    }
+    content.current.style.height = "420px";
+    footer.current.style.height = "40px";
+    input.current.style.overflow = "hidden";
+    input.current.style.height = "24px";
+  };
+
+  const sendMessage = async () => {
+    const session = authHook.session();
+    if (session !== null) {
+      await sendChatMessage(session, teamId, patientId, inputText);
+      const messages = await getChatMessages(session, teamId, patientId);
+      setMessages(messages);
+      setInputText("");
+      resetInputSize();
+      if (input.current) {
+        input.current.value = "";
+      }
+    }
+  };
+
+  const inputHandler = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(event.target.value);
+    event.target.style.height = "1px";
+    event.target.style.height = (event.target.scrollHeight) + "px";
+    if (!content.current || !footer.current) {
+      throw new Error("Cannot find elements for resize");
+    }
+    //values for scrollHeight are 24 one line, 44 2 lines and 64 3 lines
+    //we display the scrollbar only when 3 lines are present
+    if (event.target.scrollHeight > 44) {
+      event.target.style.overflow = "auto";
+      content.current.style.height = "380px";
+      footer.current.style.height = "80px";
+    } else if (event.target.scrollHeight > 24) {
+      content.current.style.height = "400px";
+      footer.current.style.height = "60px";
+      event.target.style.overflow = "hidden";
+    } else {
+      resetInputSize();
+    }
+  };
+
+  return (
+    <Card className={`${classes.chatWidget}`} >
+      <div className={`${classes.chatWidgetHeader}`}>
+        <EmailOutlinedIcon className={`${classes.icon}`}/>
+        <span className={`${classes.chatWidgetHeaderText}`}>Messages (+1)</span>
+      </div>
+      <div ref={content} id={"chatWidgetContent"} className={`${classes.chatWidgetContent}`}>
```

Comment:
opt: id="chatWidgetContent"

(no need for mustache)

Reply:
done